File: sdkwork-app-sdk-go/types/skill_package_vo.go
package types

// 技能分包信息
type SkillPackageVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	PackageId int `json:"packageId"`
	PackageKey string `json:"packageKey"`
	Name string `json:"name"`
	Summary string `json:"summary"`
	Description string `json:"description"`
	Icon string `json:"icon"`
	CoverImage string `json:"coverImage"`
	CategoryId int `json:"categoryId"`
	CategoryName string `json:"categoryName"`
	Tags []string `json:"tags"`
	Featured bool `json:"featured"`
	SortWeight int `json:"sortWeight"`
	SkillCount int `json:"skillCount"`
}

File: sdkwork-app-sdk-typescript/src/types/skill-package-vo.ts
/** 技能分包信息 */
export interface SkillPackageVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 分包ID */
  packageId?: number;
  /** 分包标识 */
  packageKey?: string;
  /** 分包名称 */
  name?: string;
  /** 分包简介 */
  summary?: string;
  /** 分包描述 */
  description?: string;
  /** 分包图标 */
  icon?: string;
  /** 封面图 */
  coverImage?: string;
  /** 分类ID */
  categoryId?: number;
  /** 分类名称 */
  categoryName?: string;
  /** 标签 */
  tags?: string[];
  /** 是否推荐 */
  featured?: boolean;
  /** 排序权重 */
  sortWeight?: number;
  /** 包含技能数量 */
  skillCount?: number;
}
